### DISCUSSION TITLE
Help with signing in my deceased wifes gmail account

### DISCUSSION THREAD

**Discussion by rwuorinen81** [community/community#65144] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Can anyone help me ?? I don't have access to her phone or her recovery email

**Comment by queenofcorgis** [comment]
Hi @rwuorinen81 ,

I am so sorry to hear about your wife's passing ❤️ . 

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Gmail and Google Products like https://support.google.com/mail/community?hl=en. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by rwuorinen81** [comment]
So she has a Gmail account I can't access as well
